controller/share_manager_controller.go

```diff
@@ -212,6 +220,42 @@ func isShareManagerPod(obj interface{}) bool {
 	return false
 }
 
+func (c *ShareManagerController) checkLeasesAndEnqueueAnyStale() error {
+	sms, err := c.ds.ListShareManagersRO()
+	if err != nil {
+		return err
+	}
+	for _, sm := range sms {
+		isStale, _, err := c.isShareManagerPodStale(sm)
+		if err != nil {
+			return err
+		}
+		if isStale {
+			c.enqueueShareManager(sm)
+		}
+	}
+
+	return nil
+}
+
+func (c *ShareManagerController) runLeaseCheck(stopCh <-chan struct{}) {
+	c.logger.Infof("Starting lease check goroutine")
+	ticker := time.NewTicker(1 * time.Second)
```

Comment:
I will put a note in the LEP test plan to call out attention to CPU load in scale testing.

Reply:
TODO: I'm thinking we can make `lease_lifetime`, `lease_check_period` and so on as global settings in the future. This change would allow some use cases to tolerate longer delinquent intervals.